packages/ui-proxy/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ui-proxy",
+  "version": "1.0.0",
+  "description": "Proxy server for Genestack frontend development",
+  "bin": {
+    "ui-proxy": "bin/ui-proxy.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "chokidar": "^2.0.4",
+    "lodash": "^4.17.4",
+    "minimist": "^1.2.0",
+    "static-server": "^2.0.4",
+    "unzip-response": "^2.0.1",
+    "ws": "^2.2.1"
+  }
+}
```

Comment:
Please, add repository.
```json
 "repository": {
    "type": "git",
    "url": "https://github.com/genestack/JS-SDK/tree/master/packages/ui-proxy"
},
```
And consider to [look around](https://github.com/genestack/JS-SDK/blob/master/packages/system-api/package.json) next time.